core/state/snapshot/wipe_test.go

```diff
@@ -30,95 +30,50 @@ import (
 func TestWipe(t *testing.T) {
```

Comment:
Any particular reason for removing the storage slot entries from this test?

Reply:
The scope of things being tested is different. Previously, the test tested more thoroughly the snapshot wiping. Since we removed the wiper, the only thing being tested is the wipeKeyRange. For wipeKeyRange, I felt that the only parts of interest were longer keys and shorter keys. 

I don't think the test becomes any better just because we test on one set of 32-byte keys and one set of 64-byte keys. 

I mean, we could even remove the whole notion of accounts/storage from this test, and just use blob data.